code/game/data_huds.dm

```diff
@@ -181,12 +184,16 @@
 //called when a carbon changes stat, virus or XENO_HOST
 /mob/living/carbon/med_hud_set_status()
 	var/image/holder = hud_list[STATUS_HUD]
+	if(istype(dna?.species, /datum/species/machine))
```

Comment:
```suggestion
	if(ismachineperson(dna?.species))
```
helper already exists for this
`#define ismachineperson(A) (is_species(A, /datum/species/machine))`